Source/CarthageKit/GitHub.swift

```diff
@@ -463,64 +464,12 @@ private func parseLinkHeader(linkValue: String) -> [(NSURL, [String])] {
 	}
 }
 
-/// Fetches the given GitHub URL, automatically paginating to the end.
-///
-/// Returns a signal that will send one `NSData` for each page fetched.
-private func fetchAllPages(URL: NSURL, _ authorizationHeaderValue: String?, _ urlSession: NSURLSession) -> SignalProducer<NSData, CarthageError> {
```

Comment:
Maybe we don't actually need this? I guess our only API usage right now is to fetch the release for a given tag, which isn't a paginated response. :flushed: